partner_communication/wizards/pdf_wizard.py

```diff
@@ -31,9 +31,15 @@ class PdfPreviewWizard(models.TransientModel):
         'partner.communication.job', required=True)
     preview = fields.Binary(compute='_compute_pdf')
     state = fields.Char(compute='_compute_state')
+    send_state = fields.Char(compute='_compute_send_state')
```

Comment:
Instead of a compute field, you can use a related field (you can change as well field state. This will simply become 
```
send_state = fields.Char(related='communication_id.state')
```
and you can remove the compute method.